src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AEADCommon.Windows.cs

```diff
@@ -5,25 +5,27 @@
 using Internal.NativeCrypto;
 using static Interop.BCrypt;
 using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
 
 namespace System.Security.Cryptography
 {
-    internal static partial class AesAEAD
+    internal static partial class AEADCommon
     {
         public static unsafe void Encrypt(
-            SafeAlgorithmHandle algorithm,
             SafeKeyHandle keyHandle,
             ReadOnlySpan<byte> nonce,
             ReadOnlySpan<byte> associatedData,
             ReadOnlySpan<byte> plaintext,
             Span<byte> ciphertext,
             Span<byte> tag)
         {
-            fixed (byte* plaintextBytes = plaintext)
-            fixed (byte* nonceBytes = nonce)
-            fixed (byte* ciphertextBytes = ciphertext)
-            fixed (byte* tagBytes = tag)
-            fixed (byte* associatedDataBytes = associatedData)
+            // Use GetNonNullPinnableReference as a workaround for VSO #32947907
```

Comment:
Bug databases change all the time.  Describing when (e.g. Win10 20H1, or ChaCha-Poly on Win10 20H2, or whatever) is fine, or just remove the comment.